### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by Clementine951**
issue template for bug and feature request

### CODE REVIEW

**Review by LucasAub — APPROVED** [review]
Looks good to me!

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (MaximeAlgosup) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Is it possible to add something to link the issue with the part of the code, to help the person who's correct the error to identify the issue quicker

**Review by MaximeAlgosup — COMMENTED** [review]
Just add something to link the issue with the part of the code, to help the person who's correct the error to identify the issue quicker

**Review by MaximeAlgosup — APPROVED** [review]
LGFM

**Pull request merged**